Automattic/vip-go-mu-plugins
# Add PHPCS Diffing

Based on @cojennin's work on Edit Flow: https://github.com/Automattic/Edit-Flow/pull/559

Generates a PHPCS report and then filters out any issues on code that wasn't touched and does not flag those.

The report is set to not fail for now until we see how well it's working.

## Checklist

CI-only change, so, the checklist isn't applicable.

## Steps to Test

- `composer install`
- `./bin/phpcs-diff.sh`

Should see something like:

```
== Generating diff against master: /tmp/tmp.6hWvZdgTSk
== Generating phpcs report: /tmp/tmp.R5DDdMYQ0U
== Running diff
100.00% Covered
```

Also, check that Circle CI's PHP 7.3 + Multisite runs this successfully (https://circleci.com/gh/Automattic/vip-go-mu-plugins/1946). Should only run for that one build.
